scripts/templates/exp_feat.rst.mako

```diff
@@ -0,0 +1,28 @@
+<%text><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+%>
+</%text>
+.. _experimental-${name}
+
+================================================================================
+${" ".join(name.split("-")).title()}
+================================================================================
+
+
+.. warning:
+
+    Experimental features:
+
+    *   May be replaced, updated, or removed at any time.
+    *   Do not require maintaining API/ABI stability of their own additions over
+        time.
+    *   Do not require conformance testing of their own additions.
+
+
+.. todo::
+
+    In the following sections you should explain and document the motivation of 
+    the experimental feature, the additions made to the specification along with
+    its valid usage.
```

Comment:
I think we should have some suggested section rather than nothing.

Reply:
👍